packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -55,9 +67,37 @@ const ShowHide = (props) => {
     preview,
     intl,
     isOpen,
+    focusRef,
     ...customProps
   } = props;
 
+  const theme = useContext(ThemeContext);
+  const contentRef = useRef(null);
+  const [containerIsActive, setContainerIsActive] = useState(false);
+
+  /**
+   * Upon showing hidden content, set activeElement to focusRef element if provided, othervise set it to content container.
+   */
+  useEffect(() => {
+    if (!isOpen) {
+      return;
+    }
+    if (focusRef?.current) {
+      focusRef?.current.focus();
+    } else if (contentRef?.current) {
+      setContainerIsActive(true);
+    }
+  }, [isOpen, focusRef, contentRef]);
+
+  useEffect(() => {
+    if (containerIsActive && isOpen && contentRef?.current) {
+      contentRef?.current?.focus();
+    }
+  }, [containerIsActive, isOpen]);
```

Comment:
Why can't this be combined with the previous useEffect?  Couldn't you just set the focus in the else if block for contentRef?